rosa_tutorials/rosa-mobb-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,336 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-load-balancer-operator"]
+= AWS Load Balancer Operator on ROSA tutorial
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-load-balancer-operator
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-12
+//---
+//date: '2023-01-03T22:07:08.574151'
+//title: AWS Load Balancer Operator On ROSA
+//aliases: ['/docs/rosa/alb-sts']
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Shaozhen Ding
+//  - Paul Czarkowski
+//---
+
+include::snippets/mobb-support-statement.adoc[leveloffset=+1]
+
+
+[TIP]
+====
+AWS Load Balancer Operator cannot be used for {product-title} Routes, thus should only be used for individual services or Ingress that does not need the full layer7 capabilties of an {product-title} route.
+====
+
+link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/[AWS Load Balancer Controller] is a controller to help manage Elastic Load Balancers for a Kubernetes cluster.
+
+* It satisfies Kubernetes link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress resources] by provisioning link:https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html[Application Load Balancers (ALB)].
+https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html[Network Load Balancers (NLB)].
+
+Compared with default AWS In Tree Provider, this controller is actively developed with advanced annotations for both link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/ingress/annotations/[ALB] and link:https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.2/guide/service/annotations/#lb-type[NLB]. Some advanced usecases are:
+
+* Using native Kubernetes Ingress with ALB
+* Integrate ALB with web application firewall (WAF)
+* Specify NLB source IP ranges
+* Specify NLB internal IP address
+
+link:https://github.com/openshift/aws-load-balancer-operator[AWS Load Balancer Operator] is used to used to install, manage and configure an instance of `aws-load-balancer-controller` in a OpenShift cluster.
+
+.Prerequisites
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/122639474/0dc07610-acc9-4ab1-b013-89b8b9fd6f54)

Reply:
These are the prereqs I was given for ALBO...not sure if they should be the same, but I would think so?